_data/sidebar_tree.yaml

```diff
@@ -75,6 +75,8 @@ tree:
     tree:
       - url: /simulator/programming/
         title: Programming
+  - url: /updates/
+    title: Updates
```

Comment:
Would it be better to nest this under the Brain Board page? After all that's what it relates to -- it's not general updates. (Note for example that the Simulator could also have updates, but I'd expect those to be on the Simulator pages).